dotCMS/src/main/java/com/dotcms/rest/BundlePublisherResource.java

```diff
@@ -45,140 +54,155 @@
      * server on this end point and if the security tokens match. If all the validations are correct the bundle will be add it<br/>
      * to the {@link PublishThread Publish Thread}.
      *
-     * @param fileName       File name to be published
-     * @param auth_token_enc Authentication token
-     * @param groupId        Group who sent the Bundle
-     * @param endpointId     End-point who sent the Bundle
-	 * @param type			 response type
-	 * @param callback 		 response callback
-	 * @param bundleName	 The name for the Bundle to publish
-	 * @param forcePush 	 true/false to Force the push
-     * @param req            HttpRequest
+     * @param fileName        File name to be published
+     * @param authTokenDigest Authentication token
+     * @param groupId         Group who sent the Bundle
+     * @param endpointId      End-point who sent the Bundle
+	 * @param type			  response type
+	 * @param callback 		  response callback
+	 * @param bundleName	  The name for the Bundle to publish
+	 * @param forcePush 	  true/false to Force the push
+     * @param request         {@link HttpServletRequest}
+	 * @param response        {@link HttpServletResponse}
      * @return Returns a {@link Response} object with a 200 status code if success or a 500 error code if anything fails on the Publish process
      * @see PublishThread
      */
     @POST
     @Path ("/publish")
     @Consumes(MediaType.APPLICATION_OCTET_STREAM)
 	public Response publish(
-			@QueryParam("FILE_NAME") String fileName,
-			@QueryParam("AUTH_TOKEN") String auth_token_digest,
-			@QueryParam("GROUP_ID") String groupId,
-			@QueryParam("ENDPOINT_ID") String endpointId,
-			@QueryParam("type") String type,
-			@QueryParam("callback") String callback,
-			@QueryParam("BUNDLE_NAME") String bundleName,
-			@QueryParam("FORCE_PUSH") final boolean forcePush,
-			@Context HttpServletRequest req
-	) {
+			@QueryParam("FILE_NAME")   final String fileName,
+			@QueryParam("AUTH_TOKEN")  final String authTokenDigest,
+			@QueryParam("GROUP_ID")    final String groupId,
+			@QueryParam("ENDPOINT_ID") final String endpointId,
+			@QueryParam("type")        final String type,
+			@QueryParam("callback")    final String callback,
+			@QueryParam("BUNDLE_NAME") final String bundleName,
+			@QueryParam("FORCE_PUSH")  final boolean forcePush,
+			@Context final HttpServletRequest  request,
+			@Context final HttpServletResponse response
+	) throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5594625)